rusk/tests/common/mod.rs

```diff
@@ -11,6 +11,7 @@ pub mod wallet;
 
 use tracing_subscriber::EnvFilter;
 
+#[allow(dead_code)]
```

Comment:
Same here on the removal.

Reply:
We use the common module for benchmarks as well and then it throws a warning that the functions and structs are not used.